No strdupa() on OpenBSD; provide the usual alloca(3)-based fallback
(same idiom as src/pipewire/utils.h) instead of a leaking strdup().

Index: spa/plugins/filter-graph/filter-graph.c
--- spa/plugins/filter-graph/filter-graph.c.orig
+++ spa/plugins/filter-graph/filter-graph.c
@@ -6,6 +6,7 @@
 
 #include <string.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <errno.h>
 #include <sys/types.h>
 #include <sys/stat.h>
@@ -30,6 +31,16 @@
 #include <spa/debug/types.h>
 #include <spa/debug/log.h>
 #include <spa/filter-graph/filter-graph.h>
+
+#ifndef strdupa
+# define strdupa(s)								      \
+	({									      \
+		const char *__old = (s);					      \
+		size_t __len = strlen(__old) + 1;				      \
+		char *__new = (char *) alloca(__len);				      \
+		(char *) memcpy(__new, __old, __len);				      \
+	})
+#endif
 
 #include "audio-plugin.h"
 #include "audio-dsp-impl.h"
